### PR TITLE
Update dashboard-slow-query.md

### PR DESCRIPTION
**Pull request by lilyjazz**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dashboard/dashboard-slow-query.md [hunk 1]
```diff
@@ -49,13 +49,33 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
 
+### SQL
+
 点击**展开** (**Expand**) 链接可以展开相应项的完整内容，点击**复制** (**Copy**) 链接可以复制完整内容到剪贴板。
 
+### 执行计划
+
+TiDB Dashboard 提供两种方式查看执行计划：图形化和文本。其中，图形化的执行计划可以让用户更清晰地了解一个语句的具体算子和对应的内容。参阅[理解 TiDB 执行计划](/explain-overview.md)文档了解如何解读执行计划。
+
+#### 如何查看图形化执行计划
```
Comment (shichun-0415) [inline dashboard/dashboard-slow-query.md:65]: 感觉不是介绍如何查看，而是如何解读图形化执行计划。
```suggestion
#### 图形化执行计划介绍
```

File: dashboard/dashboard-slow-query.md [hunk 2]
```diff
@@ -49,13 +49,33 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
+
+1. 查看顺序是从左到右，从上到下。
+2. 更上的节点是父算子，更下的节点是子算子。
+3. 不同颜色的顶栏代表算子执行在不同的组件上：黄色代表 TiDB，蓝色代表 TiKV，粉色代表 TiFlash。
+4. 节点顶栏的文本：算子的名称。
+5. 节点右侧的提示：该算子有可优化的建议。
+6. 节点下部的内容：算子的基本信息。
+7. 点击节点的弹窗：算子的全部详细信息和相关建议。
```
Comment (shichun-0415) [inline dashboard/dashboard-slow-query.md:73]: 图上没有粉色，看是不是要构造一个，重新截图。
```suggestion
- 执行计划的展示顺序是从左到右，从上到下。
- 上面的节点是父算子，下面的节点是子算子。
- 顶栏的颜色代表算子执行的组件：黄色代表 TiDB，蓝色代表 TiKV，粉色代表 TiFlash。
- 节点顶栏的文本为算子的名称。
- 节点右侧的提示为该算子的优化建议。
- 节点页面正文为算子的基本信息。
- 点击节点的弹窗，将看到算子的详细信息和相关建议。
```

**Review by time-and-fate — COMMENTED** [review]
https://docs.pingcap.com/zh/tidb/stable/dashboard-statement-details
这个页面需要加对应的文档吗？

File: dashboard/dashboard-slow-query.md (hunk 2 above)
Comment (lilyjazz) [reply dashboard/dashboard-slow-query.md:73]: 我去找找看

File: dashboard/dashboard-slow-query.md [hunk 3]
```diff
@@ -49,13 +49,33 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
 在列表中点击任意一行可以显示该慢查询的详细执行信息，包含：
 
 - SQL：慢查询 SQL 文本（下图中区域 1）
-- 执行计划：慢查询的执行计划，参阅[理解 TiDB 执行计划](/explain-overview.md)文档了解如何解读执行计划（下图中区域 2）
+- 执行计划：慢查询的执行计划（下图中区域 2）
```
Comment (shichun-0415) [inline dashboard/dashboard-slow-query.md:52]: dashboard-slow-queries-detail1.png，需要重新截图，截断文字了。

File: dashboard/dashboard-slow-query.md [hunk 4]
```diff
@@ -49,13 +49,33 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
 - 其他分类好的 SQL 执行信息（下图中区域 3）
 
 ![查看执行详情](/media/dashboard/dashboard-slow-queries-detail1.png)
 
+### SQL
+
 点击**展开** (**Expand**) 链接可以展开相应项的完整内容，点击**复制** (**Copy**) 链接可以复制完整内容到剪贴板。
```
Comment (shichun-0415) [inline dashboard/dashboard-slow-query.md:59]: 图上有两个复制按钮，Copy Formatted、Copy Original，说`点击复制链接可以复制完整内容到剪贴板`，感觉可能不太一致。

File: dashboard/dashboard-slow-query.md [hunk 5]
```diff
@@ -49,13 +49,33 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
+
+TiDB Dashboard 提供两种方式查看执行计划：图形和文本。通过图形化的执行计划，你可以更清晰地了解一个语句的具体算子和对应的内容。参阅[理解 TiDB 执行计划](/explain-overview.md)。
+
+#### 图形化执行计划介绍
+
+- 执行计划的展示顺序是从左到右，从上到下。
+- 上面的节点是父算子，下面的节点是子算子。
+- 节点顶栏的颜色代表算子执行的组件：黄色代表 TiDB，蓝色代表 TiKV，粉色代表 TiFlash。
+- 节点顶栏的文本为算子的名称。
+- 节点右侧的提示为该算子的优化建议。
```
Comment (shichun-0415) [inline dashboard/dashboard-slow-query.md:71]: 新图右侧没有提示

File: dashboard/dashboard-slow-query.md [hunk 6]
```diff
@@ -49,13 +49,33 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
+- 节点页面正文为算子的基本信息。
+- 点击节点的弹窗，将看到算子的详细信息和相关建议。
```
Comment (shichun-0415) [inline dashboard/dashboard-slow-query.md:73]: 新图没有节点的弹窗

Comment (lilyjazz) [reply dashboard/dashboard-slow-query.md:73]: 如果需要每一个内容都有显示的话，会有很多图。我都可以传上来，那麻烦考虑下如何排版好看。

Comment (lilyjazz) [reply dashboard/dashboard-slow-query.md:73]: 加了，麻烦协助调整下排版。

File: dashboard/dashboard-slow-query.md [hunk 7]
```diff
@@ -49,12 +49,32 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
+
+#### 图形化执行计划介绍
+
+- 执行计划的展示顺序是从左到右，从上到下。
+- 上面的节点是父算子，下面的节点是子算子。
+- 节点顶栏的颜色代表算子执行的组件：黄色代表 TiDB，蓝色代表 TiKV，粉色代表 TiFlash。
+- 节点的顶栏为算子的名称，正文为算子的基本信息。
+- 点击节点区域，右侧将弹出算子的详细信息。
+
+![可视化执行计划弹窗](/media/dashboard/dashboard-visual-plan-2.png)
```
Comment (Oreoxmt) [inline dashboard/dashboard-slow-query.md:73]: 是否需要放到 L66？

File: dashboard/dashboard-slow-query.md [hunk 8]
```diff
@@ -49,12 +49,32 @@ aliases: ['/docs-cn/dev/dashboard/dashboard-slow-query/']
...
+
+![可视化执行计划弹窗-侧栏](/media/dashboard/dashboard-visual-plan-popup.png)
```
[the remark this answers is not in the document]
Comment (shichun-0415) [reply dashboard/dashboard-slow-query.md:78]: Fixed in 4305d22

File: dashboard/dashboard-slow-query.md (hunk 7 above)
Comment (shichun-0415) [reply dashboard/dashboard-slow-query.md]: Fixed in 4305d22

**Comment by shichun-0415** [comment]
> ` [10f88d9](/pingcap/docs-cn/pull/10534/commits/10f88d9af39c1c02f6c89e1c0b542bbba8c457b3)`

Fixed in 10f88d9. @time-and-fate @lilyjazz PTAL

File: dashboard/dashboard-slow-query.md (hunk 6 above)
Comment (shichun-0415) [reply dashboard/dashboard-slow-query.md]: fixed

**Review by time-and-fate — APPROVED** [review]
内容上我没问题了。
不过看起来 `media/dashboard/dashboard-visual-plan-Diagnosis.png` 和 `media/dashboard/dashboard-visual-plan.png` 没有用到。

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**